tests/common/playground.rs

```diff
@@ -382,23 +391,55 @@ impl CBPlayground {
         username: String,
         password: String,
     ) -> String {
-        let uri = format!("{}/pools/default/buckets/{}/scopes", conn_string, bucket);
-
         let mut uuid = Uuid::new_v4().to_string();
         uuid.truncate(6);
         let scope_name = format!("test-{}", uuid);
 
         let mut params = HashMap::new();
         params.insert("name", scope_name.clone());
 
-        let client = reqwest::Client::new();
-        let res = client
-            .post(uri)
-            .form(&params)
-            .basic_auth(username, Some(password))
-            .send()
+        let mut client_builder = ClientBuilder::new();
```

Comment:
Seems like we should probably have a utility function for building the http client?